packages/playwright-core/src/server/chromium/chromium.ts

```diff
@@ -110,12 +110,6 @@ export class Chromium extends BrowserType {
       artifactsDir,
       downloadsPath: options.downloadsPath || artifactsDir,
       tracesDir: options.tracesDir || artifactsDir,
-      // On Windows context level proxies only work, if there isn't a global proxy
-      // set. This is currently a bug in the CR/Windows networking stack. By
-      // passing an arbitrary value we disable the check in PW land which warns
-      // users in normal (launch/launchServer) mode since otherwise connectOverCDP
-      // does not work at all with proxies on Windows.
-      proxy: { server: 'per-context' },
```

Comment:
Maybe we should keep this for a few more months since it would break connections to older Chromium and reference to `crrev.com/c/5688851` so we can remove it in 6-12 months?

Reply:
nvm, this was only to silent a check which we are removing now.